src/Parser/B3dmParser.js

```diff
@@ -54,6 +86,9 @@ export default {
      * @param {Object} options - additional properties.
      * @param {string=} [options.gltfUpAxis='Y'] - embeded glTF model up axis.
      * @param {string} options.urlBase - the base url of the b3dm file (used to fetch textures for the embeded glTF model).
+     * @param {boolean=} [options.doNotPatchMaterial='false'] - disable patchin material with logarithmic depth buffer support.
+     * @param {float} [options.opacity=1.0] - the b3dm opacity.
+     * @param {boolean|Material=} [options.overrideMaterials='false'] - override b3dm's embeded glTF materials. If overrideMaterials is a three.js material, it will material used to override.
```

Comment:
[options.material] would be a simpler naming.

> it will material used to override.

This could be rephrased to "this material will used to override existing glTF materials."

Reply:
Fixed the doc, but kept the name. I think it is important to insist on the fact that we override the B3dm material and not only define one.